Dockerfile

```
@@ -1,20 +1,11 @@
-FROM ubuntu:16.04
-
-RUN apt-get update && apt install -y software-properties-common  \
-    && add-apt-repository ppa:deadsnakes/ppa \
-    && apt update \
-    && apt-get install -y python3.7 python3-pip \
-    && apt remove -y software-properties-common && apt autoremove -y \
-    && rm -rf /var/lib/apt/lists/*
-
-# Basic env setup
-WORKDIR /opt
+FROM arterys/inference-sdk-base:0.1-cpu
```

I've since published a 1.0 release to the docker hub (same src as 0.1 just makes sense to be 1.0), so let's use that
```suggestion
FROM arterys/inference-sdk-base:1.0-cpu
```